python/cloudtik/runtime/spark/scripts/install.sh

```diff
@@ -156,8 +156,13 @@ function install_spark_with_cloud_jars() {
 }
 
 function install_hdfs_fuse() {
-    # install fuse_dfs
-    :
+    if [ -z "${HADOOP_VERSION}" ]; then
+        HADOOP_VERSION=3.3.1
+    fi
```

Comment:
HADOOP_VERSION check and initialization is not needed. The HADOOP_VERSION is set at the beginning and we would like to avoid HADOOP_VERSION=3.3.1 appearing in many places which make update HADOOP VERSION difficult.